querydsl-codegen/src/main/java/com/querydsl/codegen/EntityType.java

```diff
@@ -172,8 +180,8 @@ public Supertype getSuperType() {
         return superTypes;
     }
 
-    public String getUncapSimpleName() {
-        return uncapSimpleName;
+    public String getModifiedSimpleName() {
```

Comment:
this is not a backwards compatible change, consider deprecating the original one instead